Equinox/Globals.h

```diff
@@ -54,5 +56,68 @@ typedef unsigned int uint;
 #define RESIZABLE true
 #define TITLE "Equinox Engine Editor"
 
+inline void DrawBoundingBox(AABB& boundingBox)
```

Comment:
But this function can't be in a module if we use it from different places. Also, creating a module implies a lot more work than a global function (and more if it is inline like this one), you may not need to implement all the functions of the base class, but all those functions are called in the Game Loop, so at the end it's expensive to have it like a module, and that is not the purpose of a module.